articles/tech-resources/change-log-sparkpost-enterprise.md

```diff
@@ -6,6 +6,37 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
+
+### December, 2017
+* **Change:** Shorter SparkPost engagement tracking URLs
+  * SparkPost changed our tracking URLs to be shorter. The precise amount of change depends on how much personalization and meta data a customer is sending with their transmission.
+
+
+### November, 2017
+* **NEW:** Additional Open Pixel at message top
+  * SparkPost has added an additional open pixel to the top of each message where open tracking is selected. This is in addition to the open pixel that is normally inserted at the bottom of the message
```

Comment:
It is not an "additional open pixel at the top of each message". It is just "an open pixel at the top of each message"